### PR TITLE
update ILM integ test cluster poll interval to 1s

### PR DESCRIPTION
**Pull request by talevy**
To test a theory for https://github.com/elastic/elasticsearch/issues/35105, I am opening this 
to see how it behaves in CI.

More logging is also necessary to see how to resolve this consistently, beyond timing magic.

### CODE REVIEW

**Comment by talevy** [comment]
test this please

**Comment by talevy** [comment]
test this please

File: x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java [hunk 1]
```diff
@@ -83,8 +83,8 @@ public void testFullPolicy() throws Exception {
         // index document {"foo": "bar"} to trigger rollover
         index(client(), originalIndex, "_id", "foo", "bar");
         assertBusy(() -> assertTrue(indexExists(secondIndex)));
-        assertBusy(() -> assertFalse(indexExists(shrunkenOriginalIndex)));
         assertBusy(() -> assertFalse(indexExists(originalIndex)));
+        assertBusy(() -> assertFalse(indexExists(shrunkenOriginalIndex)));
```
Comment (gwbrown) [inline x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java]: If the order of these asserts is important in a non-obvious way, we should probably include a comment explaining why.

File: x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java [hunk 2]
```diff
@@ -151,8 +151,8 @@ public void testMoveToRolloverStep() throws Exception {
             "}");
         client().performRequest(moveToStepRequest);
         assertBusy(() -> assertTrue(indexExists(secondIndex)));
-        assertBusy(() -> assertFalse(indexExists(shrunkenOriginalIndex)));
         assertBusy(() -> assertFalse(indexExists(originalIndex)));
+        assertBusy(() -> assertFalse(indexExists(shrunkenOriginalIndex)));
```
Comment (gwbrown) [inline x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java]: If the order of these asserts is important in a non-obvious way, we should probably include a comment explaining why.

**Review by AthenaEryma — CHANGES_REQUESTED** [review]
Overall looks good, I think we should document why the order is the way it is if the order matters to timing though.

**Comment by talevy** [comment]
thank you @gwbrown, I've updated with docs!

**Comment by gwbrown** [comment]
While those docs are good, what I meant was something more like:
```
// These asserts are in the order that they should be satisfied in, in order to maximize the time for all operations to complete. 
// An "out of order" assert here may result in this test occasionally timing out and failing inappropriately.
```

Sorry I wasn't clear!

**Comment by talevy** [comment]
thank you @gwbrown. added in the extra context you mentioned about ordering 
alongside the existing!

**Review by AthenaEryma — APPROVED** [review]
Looks good, thanks!

**Pull request merged**